Content.Server/Administration/Commands/PlayGlobalSound.cs

```diff
@@ -21,6 +21,23 @@ public sealed class PlayGlobalSound : IConsoleCommand
     public void Execute(IConsoleShell shell, string argStr, string[] args)
     {
         Filter filter;
+        var audio = AudioParams.Default;
+        var volumeOffset = 0;
+
+        // Try to specify a new volume to play it at.
+        if (args.Length > 1)
+        {
+            if (int.TryParse(args[1], out var volume))
+            {
+                audio = audio.WithVolume(volume);
+                volumeOffset = 1;
+            }
+            else
+            {
+                shell.WriteError(Loc.GetString("play-global-sound-command-volume-parse", ("volume", args[1])));
+                return;
+            }
+        }
```

Comment:
This can be moved to the `default` switch case below, since it will always execute on `args.Length > 1`